content/knowledge-base/install-unity-version.md

```diff
@@ -0,0 +1,117 @@
+
+
+## Environment variables
+
+You will need to a three environment variables in your codeamagic.yaml called  `UNITY_VERSION`, `UNITY_VERSION_CHANGSET`, and `UNITY_VERSION_BIN`. 
+
+You should set the value of `UNITY_VERSION` to the full version number as display on the Unity Hub download archive page. For example, if the version is 2019.4.38f1, set the variable as follows:
+
+`UNITY_VERSION: 2019.4.38f1`
+
+The value of `UNITY_VERSION_CHANGESET` should be set using the **changeset** id that you obtained from the release notes page on the Unity Hub download archive. For example, if the changeset id is fdbb7325fa47 then set the variable as follows:
+
+`UNITY_VERSION_CHANGESET: fdbb7325fa47`
+
+The `UNITY_VERSION_BIN` should be set as follows so the Unity binary path is declared for the version you want to build with:
+
+`UNITY_VERSION_BIN: /Applications/Unity/Hub/Editor/${UNITY_VERSION}/Unity.app/Contents/MacOS/Unity`
+
+
+## Activating Unity
+
+Even though you are installing a different version of Unity to build your apps with, you should use the default Unity version that is already installed on the machine. Unity Hub CLI commands not work correctly if a license is not already active on the machine.
+
+
+## Install Unity script
+
+After activating the Unity license as usual, add the following script to install the desired version and modules you wish to use. The example below uses Unity Hub CLI commands to install the specified Unity version as well as the Android and iOS Build Support modules.
+
+```yaml
+    - name: Install Unity version
+      script: | 
+        /Applications/Unity\ Hub.app/Contents/MacOS/Unity\ Hub -- --headless install --version $UNITY_VERSION --changeset $UNITY_VERSION_CHANGESET 
+        /Applications/Unity\ Hub.app/Contents/MacOS/Unity\ Hub -- --headless install-modules --version $UNITY_VERSION -m ios android 
+```
+
+
+## Building with the newly installed Unity version
+
+Use the Unity version you installed on the machine:
+
+```yaml
+    - name: Build the Xcode project
+      script: | 
+        $UNITY_VERSION_BIN -batchmode -quit -logFile -projectPath . -executeMethod BuildScript.$BUILD_SCRIPT_IOS -nographics -buildTarget iOS
+```
+
+## Android Workflow configuration sample
+
+```yaml
+workflows:
+  unity-android-workflow:
+      name: Unity Android Workflow
+      instance_type: mac_pro
+      max_build_duration: 120
+      environment:
+        groups:
+          # Add the group environment variables in Codemagic UI (either in Application/Team variables) - https://docs.codemagic.io/variables/environment-variable-groups/
```

Comment:
```suggestion
          # Add the group environment variables in Codemagic UI (in Application or Team variables) - https://docs.codemagic.io/variables/environment-variable-groups/
```

I've always had a problem with this sentence as it's missing the "or" in the either/or construct.